packages/jest-runtime/src/index.js

```diff
@@ -618,14 +618,20 @@ class Runtime {
     from: Path,
   ) {
     switch (moduleName) {
+      case 'async_hooks': // Pure native module.
       case 'buffer': // Causes issues when passing buffers to another context.
       case 'module': // Calls into native_module, which is not mockable.
-      case 'os': // Pure native module.
         // $FlowFixMe: dynamic require needed.
         localModule.exports = require(moduleName);
         break;
 
-      case 'process':
+      case 'os': // Pure native module.
+      case 'v8': // Contains invalid references.
+        // $FlowFixMe: dynamic require needed.
+        localModule.exports = deepCyclicCopy(require(moduleName));
```

Comment:
Does it have a noticeable impact on perf? And are we sure there's nothing to blacklist there?

Reply:
I checked both; `v8` is just a set of functions, and `os` has some additional properties (`os.constants`), but it simple enough to be copied.